frontend/src/pages/MainPage.tsx

```diff
@@ -0,0 +1,11 @@
+import "./MainPage.css"
+import {content} from "../consts/content.ts";
```

Comment:
TS2307: Cannot find module ../ consts/ content. ts or its corresponding type declarations.

12:33:46 [vite] Internal server error: Failed to resolve import "../consts/content.ts" from "src/pages/MainPage.tsx". Does the file exist?
  Plugin: vite:import-analysis
  File: C:/Programming/kubernetes-access-manager/frontend/src/pages/MainPage.tsx:2:22
  16 |  }
  17 |  import "./MainPage.css";
  18 |  import { content } from "../consts/content.ts";
     |                           ^
  19 |  export const MainPage = () => {
  20 |    return /* @__PURE__ */ jsxDEV(Fragment, { children: content }, void 0, false, {
      at TransformPluginContext._formatError (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:49193:41)
      at TransformPluginContext.error (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:49188:16)
      at normalizeUrl (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:63984:23)
      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
      at async file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:64116:39
      at async Promise.all (index 4)
      at async TransformPluginContext.transform (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:64043:7)
      at async PluginContainer.transform (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:49034:18)
      at async loadAndTransform (file:///C:/Programming/kubernetes-access-manager/frontend/node_modules/vite/dist/node/chunks/dep-CDnG8rE7.js:51867:27)